### DISCUSSION TITLE
JSONs for all verses

### DISCUSSION THREAD

**Discussion by samanyougarg** [vedicscriptures/bhagavad-gita-api#8] [General]
Hare Krishna @PtPrashantTripathi,

Love the work you have been doing with the Bhagavad Gita API.

We at the Gita Initiative are working on creating interactive digital experiences that encourage and challenge people to practice the presence of God daily.

Currently, we are working on v2 of BhagavadGita.io (which aims to be the best Bhagavad Gita app) and were wondering if you could provide us the JSONs for the various translations and commentaries being used by your API so we can offer the users with multiple options.

Also, if you would be interested in joining forces, please join our Discord group: https://discord.gg/4V3VrqHyZx

**Comment by PtPrashantTripathi** [comment]
Radhe Radhe @samanyougarg ,
Mail me a proposal on vedic.scriptures@outlook.com

**Reply by samanyougarg** [reply]
Sure, will email today.

**Comment by sagarbhosale** [comment]
Hi all, I am writing an app for readers of the Gita too. I am planning to keep it really simple and elegant. Mostly plain text and features like bookmarks and social sharing (and a few more features I have in mind).

I am curious about the app @samanyougarg is building. I would love to help spread the divine word thru technology. Let's join forces, if you are interested.

The data is great! But it can use some clean up. For example: there are sections where a comment refers to another comment. It would be great to have links to the referred comment.

**Reply by sagarbhosale** [reply]
Another example where a clean up would be great is Abhinav Gupta's commentary on 1.3.

"1.2  1.9  Why this exhaustive counting? The reality of things is this:"

Seems incomplete to me. How can we do this better?

**Reply by samanyougarg** [reply]
Hi @sagarbhosale, yes would love to join forces.

Please join our discord group https://discord.gg/4V3VrqHyZx. We also have a bi-weekly call on Saturdays, the next one being tomorrow. Would love for you to join that as well.
